### ISSUE TITLE
FLIR MFD TOF readout is 0

### ISSUE THREAD

**Issue by ampersand38** [Project-Hatchet/public_h-60#59]
**Expected behavior**
TOF should change based on range.

https://youtu.be/g3cqKMYDoF8

**Comment by FatLurch** [comment]
This is intended to be TOF for the weapons, correct?

I had looked into trying to compute TOF for weapons as part of Turret Enhanced and it got pretty screwy for the various Hellfire modes.

**Comment by FatLurch** [comment]
I looked at the config: it seems like this is setup correctly. We're using the MissileFlightTime source from https://community.bistudio.com/wiki/Arma_3:_Multi-Function_Display_(MFD)_config_reference#Float

I was able to get a TOF other than 0 only in specific circumstance:

1. Missile locked onto a target (I was using laser)
2. Target at significant range (>1 km)
3. Target within the PIP draw distance

At 5.7 km using the H-60 AGM-114K I had a TOF listed as 18.

Even at that, the TOF was really buggy for me. I'd have to re-lock multiple times to get it to display a TOF.
